resources/texts/change_log.txt

```diff
@@ -1,3 +1,65 @@
+- Cheetah Gcode Flavor: Introduced a new Marlin-like Gcode flavor called Cheetah.
+- SpaceMouse support: Interact with your object in 3D thanks to a collaboration with 3Dconnexion
+- Cloud Printing for Sketch Sprint: Enabled printing over cloud with Digital Factory for Sketch Sprint users.
+- Interlocking Settings: Moved Interlocking settings out of experimental into expert
+- Build System Upgrade: Upgraded the build system from Conan 1 to Conan 2. Updated documentation is available.
+- Preview Looping: When the last layer is played in the preview, the first layer will now play again instead of stopping.
+- Updated About Page: The About Page now shows the used sources, their licenses, and their versions in a clearer way
+- Flip Y-axis Translate Tool Handle: Added an option to flip the Y-axis translate tool handle in preferences contributed by @GregValiant.
+- Rotation by Input & Snap Angle Input: Introduced rotation by input & snap angle input contributed by @HellAholic and @GregValiant.
+- Purge Lines And Unload Filament Post Processing Script: Added a Purge Lines and Unload Filament Post Processing Script contributed by @GregValiant and @HellAholic
+- Thingiverse "Open in Cura" Button Linux Support: Enabled the "Open in Cura" button from Thingiverse to open files in Linux contributed by @hadess.
+- Multitool Printer Configuration Options: Introduced 3 new configuration options in machine settings for multitool printers contributed by @TheSin-.
+- Search and Replace Post-Processing Plug-In: Significantly improved the Search and Replace post-processing plug-in with features like replacing only the first instance, limiting search to a layer range, and ignoring start-up or ending G-code contributed by @GregValiant
+- Enabled Relative extrusion (M82 and M83 commands) for Marlin-flavored GCode, contributed by @EmJay276
+
+
+* New settings:
+- Overhanging Wall Speeds, now gives you the ability to tune multiple values. Don’t forget to adjust the Overhanging Wall Angle to start using the setting. 
+- Minimum Layer Time with Overhang and Minimum Overhang Segment Length: Fine-tune the minimum layer time for overhangs.
+- Distance to Walls: Finetune combing movements.
+- Overhanging Wall Speeds: Fine-tune speeds based on different overhang percentages.
+- Pressure Advance Factor Setting: New setting for machine definitions.
+- You can now tune the Bottom Surface Skin, like you can tune the Top Surface Skin! You can now tune Extruder, Layers, Line Width, Pattern, Line Directions, Outer Wall Flow, Inner Wall(s) Flow, Flow, Outer Wall Speed, Inner Wall Speed, Skin Speed, Inner Wall Acceleration, Skin Acceleration, Outer Wall Jerk, Inner Wall Jerk, Skin Jerk, and Monotonic Bottom Surface Order
+- Enable/Disable USB Printing: A preference setting to indicate that you are using the printer over USB functionality. This setting lays the groundwork for automatically disabling USB printing in the next release when it’s not being used.
```

Comment:
We should say that this is for now a hidden setting, or don't mention it, nothing is actually displayed anywhere so this text gives the impression that people can find a checkbox somewhere.